src/components/Home/Home.jsx

```diff
@@ -1,27 +1,73 @@
 import React from 'react';
```

Comment:
with version 17 of react you don't need to import in every file React, so, you can remove this line :D

Reply:
It has been solve, thanks